README.md

```diff
@@ -2,80 +2,129 @@
+- Project setup and installation can be easily handled thanks to the integration with the [Task Runner][4] project.
+- All system requirements are containerized using [Docker Composer][5]
+
+### Project setup
+
+Developing the theme requires a local copy of ECL assets, including Twig templates, SASS and JavaScript source files. 
+
+In order to fetch the required code you'll need to have [Node.js (>= 8)](https://nodejs.org/en) installed locally.
+
+To install required Node.js dependencies run:
 
-Build the artifacts:
 ```
-$ npm run build
+$ npm install
 ```
 
-Install all the composer dependencies:
+To build the final artifacts run:
 
 ```
-$ composer install
+$ npm run build
 ```
 
-Customize build settings by copying `runner.yml.dist` to `runner.yml` and
-changing relevant values.
+This will compile all SASS and JavaScript files into self-contained assets that are exposed as [Drupal libraries][11].
 
-Setup test site by running:
+In order to download all required PHP code run:
 
 ```
-$ ./vendor/bin/run drupal:site-setup
+$ composer install
 ```
 
-This will symlink the theme in the proper directory within the test site and
-perform token substitution in test configuration files.
+This will build a fully functional Drupal site in the `./build` directory that can be used to develop and showcase the
+theme.
 
-Install test site by running:
+Before setting up and installing the site make sure to customize default configuration values by copying [runner.yml.dist](runner.yml.dist)
+to `./runner.yml` and override relevant properties.
+
+To setup the project run:
 
 ```
-$ ./vendor/bin/run drupal:site-install
+$ ./vendor/bin/run drupal:site-setup
 ```
 
-Your test site will be available at `./build`.
-
-## Disable Drupal 8 caching during development
+This will:
 
-Manually disabling Drupal 8 caching is a laborious process that is well described [here](https://www.drupal.org/node/2598914).
+- Symlink the theme in  `./build/themes/custom/oe_theme` so that it's available to the target site
+- Setup Drush and Drupal's settings using values from `./runner.yml.dist`
+- Setup PHPUnit and Behat configuration files using values from `./runner.yml.dist`
 
-Alternatively you can use the following Drupal Console command to disable/enable Drupal 8 caching:
+After a successful setup install the site by running:
 
 ```
-$ ./vendor/bin/drupal site:mode dev  # Disable all caches.
-$ ./vendor/bin/drupal site:mode prod # Enable all caches.
+$ ./vendor/bin/run drupal:site-install
 ```
 
-Note: to fully disable Twig caching the following additional manual steps are required:
+This will:
 
-1. Open `./build/sites/default/services.yml`
-2. Set `cache: false` in `twig.config:` property.
-3. Rebuild Drupal cache: `./vendor/bin/drush cr`
-
-This is due to the following [Drupal Console issue](https://github.com/hechoendrupal/drupal-console/issues/3854).
+- Install the target site
+- Set the OpenEuropa Theme as default theme
```

Comment:
In this particular context, I think that adding article could be considered ('as **the** default theme').